pupil-spa/.vscode/launch.json

```diff
@@ -0,0 +1,15 @@
+{
+  // Use IntelliSense to learn about possible attributes.
```

Comment:
I am surprised that this `//comment is allowed in a JSON file.

Reply:
it will no doubt be auto generated, as it's VsCode specific, which has a file type called 'JSON with comments'